### PR TITLE
Run app engine dev server in debug mode via the app tools lib (2)

### PR DESCRIPTION
**Pull request by nbashirbello**
Had some branch/merge problems with first PR (#286)

### CODE REVIEW

**Comment by nbashirbello** [comment]
@elharo @briandealwis

**Comment by elharo** [comment]
LGTM

File: plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerBehaviour.java [hunk 1]
```diff
@@ -115,6 +109,60 @@ void startDevServer(List<File> runnables, MessageConsoleStream stream) {
...
+      Activator.logError("Error starting server: " + ex.getMessage());
+      stop(true);
+    }
+  }
+
+  private void initializeDevServer(MessageConsoleStream stream) {
+    LocalAppEngineOutputLineListener outputListener =
+        new LocalAppEngineOutputLineListener(stream);
+
+    CloudSdk cloudSdk = new CloudSdk.Builder()
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerBehaviour.java]: Use `com.google.cloud.tools.eclipse.sdk`'s `CloudSdkProvider#createBuilder()` to create the builder.

Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerBehaviour.java]: Forget my comment above: we should have a way to configure path resolver helpers with `CloudSdk`.  I wonder if we can do something with `ServiceLoader`…

**Pull request merged**